lib/pdk/validators/base_validator.rb

```diff
@@ -44,19 +46,26 @@ def self.invoke(report, options = {})
         return 0 if targets.empty?
 
         PDK::Util::Bundler.ensure_binstubs!(cmd)
-        cmd_argv = parse_options(options, targets).unshift(cmd_path)
-        cmd_argv.unshift('ruby', '-W0') if Gem.win_platform?
 
-        command = PDK::CLI::Exec::Command.new(*cmd_argv).tap do |c|
-          c.context = :module
-          c.add_spinner(spinner_text)
-        end
+        targets = (self::INVOKE_STYLE == :per_target) ? targets.combination(1).to_a : Array[targets]
```

Comment:
I don't follow how the invoke style actually changes here. For a validation of _*.json_ , I think the two possible outcomes for _targets_ here are:

    [['a.json'],['b.json'],['c.json']]

or

    ['a.json', 'b.json', 'c.json']

But in the code below, in both cases, _targets.each_ just iterates on each of the 3 in turn?